api/http_internal.go

```diff
@@ -113,6 +116,9 @@ func NewCatalystAPIRouterInternal(cli config.Cli, vodEngine *pipeline.Coordinato
 	// Handler for USER_NEW triggers
 	broker.OnUserNew(accessControlHandlers.HandleUserNew)
 
+	// Handler for USER_END triggers. Only do this if we have a database
```

Comment:
I assume this is implemented in the handler, not really pertinent information
```suggestion
	// Handler for USER_END triggers.
```

Reply:
Good catch, it _was_ here and then I realised it made more sense inside